backend/src/main/java/pl/edu/agh/awiteks_backend/utilities/ForumUtilities.java

```diff
@@ -12,7 +12,7 @@ public class ForumUtilities {
     public List<UserForumThreadsSummary> getNewestForumPostsFromFollowedThreads(User user) {
         return user.getFollowedThreads().stream().limit(LIMIT).map(forumThread -> {
             forumThread.getForumPosts().sort(Comparator.comparing(ForumPost::getDate));
-            final var forumPost = forumThread.getForumPosts().stream().findFirst().orElseThrow();
+            final var forumPost = forumThread.getForumPosts().stream().reduce((first, second) -> second).orElseThrow();
```

Comment:
W sumie to nawet nie trzeba był sortować listy, jak tylko wybieramy najnowszy element, to używamy funkcji `.min()`, lub `.max()` z odpowiednim `Comperatorem` zamiast `.sort()`, ale niech już tak zostanie